omui/src/components/drawer/drawer.tsx

```diff
@@ -8,16 +8,21 @@ import {
   DrawerContent,
   DrawerCloseButton,
   DrawerProps as IDrawerProps,
-  UseDisclosureProps
+  UseDisclosureProps as IUseDisclosureProps
 } from '@chakra-ui/core';
 
+type UseDisclosureProps = IUseDisclosureProps & {
```

Comment:
You shouldn't need to do this, do you really need to? The only disclosure props are isOpen, onClose, onOpen, and onToggle: https://chakra-ui.com/usedisclosure#return-value